pkg/object/object.go

```diff
@@ -45,9 +48,27 @@ type OpenCompose struct {
 	Volumes  []Volume
 }
 
+func (s *Service) Validate() error {
+	// validate service name, like it cannot have underscores, etc.
+
+	// validate containers
+
+	// validate replicas
+	if s.Replicas != nil && *s.Replicas < 0 {
```

Comment:
see my other comment regarding docs (https://github.com/redhat-developer/opencompose/pull/66/files#r110106733)
than if you squash it, i think we can merge it